src/firefly/js/visualize/PlotChangeTask.js

```diff
@@ -322,7 +322,8 @@ function processPlotReplace(dispatcher, result, pv, makeSuccessAction, makeFailA
             var overlayPlotViews = [];
             resultAry.forEach((r, i) => {
                 if (i === 0) return;
-                var plot = WebPlot.makeWebPlotData(pv.plotId, r.data[WebPlotResult.PLOT_CREATE], true);
+                const {imageOverlayId}= pv.overlayPlotViews[i-1];
+                var plot = WebPlot.makeWebPlotData(imageOverlayId, r.data[WebPlotResult.PLOT_CREATE][0], {}, true);
                 overlayPlotViews[i - 1] = {plot};
             });
```

Comment:
just would like to confirm that that resultant length of overPlotViews is one less that of resultAry.